### ISSUE TITLE
Learning to Learn with Generative Models of Neural Network Checkpoints

### ISSUE THREAD

**Issue by reyllama** [reyllama/paper-reviews#203]
# TL; DR
- Train a generative model that is capable of generating model weights (essentially a hyper network) given initializations, starting loss/error/return and target loss/error/return.

# Motivation

- Manual optimization-based methods (typical gradient descent) cannot leverage past training experiences (aside from initialization) to render training more efficient.
- 

# Formulation

# Evaluation

# Inspiration
